lib/index.js

```diff
@@ -535,7 +548,7 @@ api.isDuplicateError = err => (api.isDatabaseError(err) &&
  *
  * @return true if the error is a duplicate key error, false if not.
  */
-api.isDatabaseError = err => (err && err.name === MDBE_ERROR);
+api.isDatabaseError = err => (err && MDBE_ERRORS.includes(err.name));
```

Comment:
If this is changing -- we'll need to change the docs (see above that it refers only to a duplicate key error) and check every place it is used.

Reply:
Just so you know the original error code `MDBE_ERROR` is not exclusive to a DuplicateKey error not to mention that `isDatabaseError` has all types of weird naming conflicts if intended solely to catch a duplicate key error so the docs were probably wrong to start with. Changed it here: https://github.com/digitalbazaar/bedrock-mongodb/pull/33/commits/3a25e58b4407b17872aa270c83a7e712cd28b2f8